docs/file-formats/xml-data-file.md

```diff
@@ -980,6 +945,47 @@ Technical note: A `<link>` is in fact a `<resource restype="LinkObj">`. But it i
 shortcut, so that the XML file can be validated more precisely.
 
 
+### `<video-segment>` / `<audio-segment>`
+
+`<video-segment>` and `<audio-segment>` represent sections of an audio/video file. 
+They can have the following predefined properties:
+
+- `isSegmentOf` (1)
+- `hasSegmentBounds` (1)
+- `hasTitle` (0-1)
+- `hasDescription` (0-n)
+- `hasComment` (0-n)
+- `hasKeyword` (0-n)
+- `relatesTo` (0-n)
+
+The bounds of the segment are defined by two floating point numbers, which represent the seconds.
```

Comment:
say what floating point is. What happens with milliseconds?

Perhaps you could write:
The segment bounds must be entered in seconds. Decimal values for milliseconds are allowed. eg. 1.4:2